src/tests/functional/shared_test_classes/src/base/utils/compare_results.cpp

```diff
@@ -8,6 +8,7 @@
 #include "ov_ops/augru_cell.hpp"
 #include "ov_ops/augru_sequence.hpp"
 #include "ov_ops/rms.hpp"
+#include "openvino/op/paged_attention.hpp"
```

Comment:
Whys is required?

Reply:
Problem was that functional test param generator and comparator seem to require op to be registered in `opset_private_tbl.hpp`, and without that include the build would fail. I think that was also motivation for includes from `ov_ops/*.hpp`.
I'm aware that this solution is not ideal (mixing with transformation internal ops), but this was simplest solution to just enable tests to execute. Better solution would probably be to use `opset_int_tbl.hpp` from template plugin

Reply:
It was not clear why added here as only include added without any code.